packages/client-sdk-web/src/vector-client-props.ts

```diff
@@ -0,0 +1,12 @@
+import {CredentialProvider, Configuration} from '.';
+
+export interface VectorClientProps {
+  /**
+   * Configuration settings for the vector client
+   */
+  configuration: Configuration;
```

Comment:
@pgautier404 I think this config object has a bunch of stuff like retries on it that we may want to handle differently for the Vector client. @malandis and I had this same discussion on his Python PR, and I think we are leaning towards having a separate Configuration object for Vector clients so that they can evolve independently.

We got burned a bit by copy/pasting too much of the config/wrapper stuff from CacheClient to TopicClient and AuthClient early on, and we kinda tried to unwind some of that. e.g. we have a separate config object for Topics now. so I want to make sure we're making a deliberate and forward-compatible choice here.

lmkwyt

Reply:
That works for me-- I can imagine we might want to add distinct config for each type.

Reply:
Added in 0cc099d.